ingestion/functions/parsing/mexico/mexico.py

```diff
@@ -18,26 +18,34 @@
     import parsing_lib
 
 
-with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), "dictionaries.json")) as json_file:
+with open(
+    os.path.join(os.path.dirname(os.path.abspath(__file__)), "dictionaries.json")
+) as json_file:
     maps = json.load(json_file)
 
-_COMORBIDITIES_DICT = maps['comorbidities']
+_COMORBIDITIES_DICT = maps["comorbidities"]
 
-_STATES = maps['states']
+_STATES = maps["states"]
 
-_MUNICIPALITIES = maps['municipalities']
+_MUNICIPALITIES = maps["municipalities"]
 
 
 def convert_location(state_code: str, municipality_code: str):
     """
     Codes beginning with 9 denote missing information
```

Comment:
Documenting this is good, but the docstring for the function shouldn't (just) be this. Things I might do:

 - fill out the docstring, so that this message is a note at the end of a description of the function's behaviour
 - reorganise the code to make the intent clearer:
```python
missing_data_prefix = "9"
if !municipality_code.startswith(missing_data_prefix):
```